movie-react-app/src/components/SearchPage.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+
+class SearchPage extends Component {
+  constructor(props) {
+    super(props);
+    this.onSearch = this.onSearch.bind(this);
+    this.renderMovie = this.renderMovie.bind(this);
+    this.state = {
+      searchText: "",
+      results: []
+    };
+  }
+
+  onSearch() {
+    fetch(
+      `http://www.omdbapi.com/?apikey=aecbec56&s=${encodeURIComponent(
```

Comment:
I've hardcoded my API key into the URL for now but probably we want to move this out